cloud-secrets.js

```diff
@@ -0,0 +1,47 @@
+const {SecretManagerServiceClient} = require('@google-cloud/secret-manager');
```

Comment:
It kind of feels like we should start exploring an option where we can have a shared place for things like this and other transforms/plugins/etc that we share between web.dev and d.c.c.

Reply:
Maybe create a public repo with shared code and just `npm` install that?